### PR TITLE
Added a section to list contributors

### PR DESCRIPTION
**Pull request by marissap**
- Update index.html with a new section
- Added scss file for styles for this section
- Updated README.md with steps for future contributors to add their name

### CODE REVIEW

**Comment by marissap** [comment]
Work for issue #23

File: index.html [hunk 1]
```diff
@@ -305,6 +305,35 @@ <h2 class="section-title mt-0">Stay in the know</h2>
...
+                    <div class="text-center">
+                        <h2 class="section-title mt-0">Our Dream Team</h2>
+                        <div class="contributors-wrapper">
+                            <div class="contributors-row">
+                                <div class="contributors-column technical">
+                                    <svg height="48" id="processor" viewBox="0 0 48 48" width="48" xmlns="http://www.w3.org/2000/svg"><defs></defs><rect class="vi-primary" height="24" width="24" x="12" y="12"/><path class="vi-accent" d="M16,16H32V32H16V16ZM8,16h4v3H8V16Zm0,6h4v3H8V22Zm0,6h4v3H8V28ZM36,16h4v3H36V16Zm0,6h4v3H36V22Zm0,6h4v3H36V28ZM32,8v4H29V8h3ZM26,8v4H23V8h3ZM20,8v4H17V8h3ZM31,36v4H28V36h3Zm-6,0v4H22V36h3Zm-6,0v4H16V36h3Z"/></svg>
+                                    <div class="contributor-list">
+                                        <p>Ashley Narcisse</p>
+                                        <p>Takashi Wickes</p>
+                                        <p>Blake Pfaff</p>
```
Comment (madiedgar) [inline index.html]: would love for these to be anchor tags to link to the contributor's github! 🎁 (adding this comment to the `contributing.md` file as well)

File: README.md [hunk 2]
```diff
@@ -64,6 +64,26 @@
...
+
+**3️⃣ Did you do technical work or non-technical?**
+If was technical put your name under the column with the chip svg icon!
+```
+<div class="contributors-column technical">
+```
+If was non-technical put your name under the column with the lightbulb svg icon!
+```
+<div class="contributors-column non-technical">
+```
```
Comment (madiedgar) [inline README.md]: This is a great way of helping contributors understand what to do! Could you add in a little more detail, as some of our contributors don't know html. 😄 Thinking of something along these lines:

``` html 
<div class="contributors-column technical">
  ...
  <a href="https://github.com/YOURUSERNAME">YOUR NAME</a> 
  ...
</div>
```

File: index.html (hunk 1 above)
Comment (madiedgar) [reply index.html]: Ashely Nacrisse: https://github.com/darkfadr
Takashi Wickes: https://github.com/takashiw
Blake Pfaff: https://github.com/blakecarson

Comment (marissap) [reply index.html]: Are they technical or non-technical contributors?

File: README.md (hunk 2 above)
Comment (marissap) [reply README.md]: Did we want to take this out of `README.md` and create `contributing.md`?

File: index.html (hunk 1 above)
Comment (madiedgar) [reply index.html]: We're going to be keeping track on the README the types of contributions people are making according to [`allcontributors`](https://allcontributors.org/docs/en/emoji-key). These are their types of contributions! ✨ 

Ashley: `🚇 Infrastructure`
Takashi: `🎨 Design`
Blake: ` 📖 Documentation`

**Comment by madiedgar** [comment]
@marissap Wow, incredible work! Thank you so much, for being thoughtful and creative. 😄 

Would love to honor the time you put into this pull request before I ask for additional changes. ❤️  I'm going to merge this pull request onto the `all-contributors` branch of `legesher-dot-io` with one minor change: could you take out the additions in the README file for the time being (creating separate issue for that 😁).

I'm going to create more issues describing the details of the additional changes. As you already put so much time and thought to these items, I'd love to give you first dibs. 😄 (also make sure to sign up on [Hacktoberfest](hacktoberfest.digitalocean.com) so you get credit for the PR's 🎉)

Here's a preview of the additional issues:
- **Add a `contributing` page** to the website so all of the repositories can link to the same page. It would help with guidelines and different ways of contributing across the project. 
- **Incorporating the [`allcontributors`](https://allcontributors.org/docs/en/emoji-key) emoji key** for contributing. Currently thinking about a fun, creative way to organize all the different categories of contributions 🤔.

Thank you again!! 👏

**Comment by marissap** [comment]
@madiedgar I attempted to remove the `README.md` changes and add the Github links to the contributors but I think I messed up on the branches🙃Could you help me sort that out? I don't want to make more work for you!!

**Comment by madiedgar** [comment]
I think that might have been my doing @marissap when changing the base branch. It looks good to me though! 👍 Great work 😄 I'm going to merge these changes and then draft up the next two issues for where this page should go.

**Comment by marissap** [comment]
Thank you @madiedgar ✨It was super fun helping out with this!❤️

**Pull request merged**